.github/workflows/ci.yml

```diff
@@ -1,6 +1,10 @@
 name: CI
 
-on: [push, pull_request]
+on:
+  push:
+    branches:
+    - master
+  pull_request:
```

Comment:
is this valid yaml?
```suggestion
  pull_request: {}
```

Reply:
seems so, just copied it from the link. But the empty dict looks nicer :ok_hand: